deploy/dnsimple/templates/deployment.yaml

```diff
@@ -24,14 +24,14 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ include "dnsimple-webhook.fullname" . }}
+      {{- if .Values.image.pullSecret }}
+      imagePullSecrets:
+        - name: {{ .Values.image.pullSecret }}
+      {{- end }}
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          {{- if .Values.image.pullSecret }}
-          imagePullSecrets:
-            - name: {{ .Values.image.pullSecret }}
-          {{- end }}
```

Comment:
This wasn't really in the scope of this PR but I needed to fix this so that I could test with a privately hosted build of the custom webhook image. `imagePullSecrets` should be specified under `spec` and not under `spec.containers...` - https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-pod-that-uses-your-secret